fdbserver/storageserver.actor.cpp

```diff
@@ -3475,6 +3567,29 @@ ACTOR Future<Void> checkBehind( StorageServer* self ) {
 	}
 }
 
+ACTOR Future<Void> reportStorageServerState(StorageServer* self) {
+	if (!SERVER_KNOBS->REPORT_DD_METRICS) {
+		return Void();
+	}
+
+	loop {
+		wait(delay(SERVER_KNOBS->DD_METRICS_REPORT_INTERVAL));
+
+		const auto numRunningFetchKeys = self->currentRunningFetchKeys.numRunning();
+		if (numRunningFetchKeys == 0) {
+			continue;
+		}
+
+		const auto longestRunningFetchKeys = self->currentRunningFetchKeys.longestTime();
+		TraceEvent(SevInfo, "FetchKeyCurrentStatus")
```

Comment:
Maybe we can change to `SevWarnAlways` if the longest running time is longer than the threshold defined in `logFetchKeysWarning()`.